README.md

```diff
@@ -21,7 +21,9 @@ This starter template lets you quickly start working with the [BeeAI Framework](
 1. Clone this repository or [use it as a template](https://github.com/new?template_name=beeai-framework-py-starter&template_owner=i-am-bee).
 2. Install dependencies `poetry install`.
 3. Install and start the poetry environment `poetry self add poetry-plugin-shell` and `poetry shell`
-4. Configure your project by filling in missing values in the `.env` file (default LLM provider is locally hosted `Ollama`).
+4. Configure your project by 
```

Comment:
The English is a bit off here: I would either update this to 
```suggestion
4. Configure your project:
```
or updating the following lines to "making", "renaming", and "filling" for correct tense